packages/main/src/resolvers/conversation.test.ts

```diff
@@ -638,6 +638,116 @@ describe("when addressing conversations", () => {
     })
   })
 
+  it("an edited message will have an isRead field", async () => {
```

Comment:
Think of `it` as the first word in the sentence that is the test description. And what does it mean that an edited message will have an isRead field? All presentable elements have an isRead field, regardless of whether they have been edited.

What I think you want to specify in this test is how `isRead` is computed in the case of an edited message. Is the presentable considered to be read if the latest edit is read? Or if the original revision is read? Or only if all revisions have been read? In this test it looks like both the original revision and the edited revision are unread, so the test does not answer that question.